libs/sdk-ui-filters/src/AttributeFilterLoader/internal/store/common/sagas.ts

```diff
@@ -0,0 +1,58 @@
+// (C) 2022 GoodData Corporation
+
+import { SagaIterator } from "redux-saga";
+import { call, cancelled, getContext } from "redux-saga/effects";
+
+import { AttributeFilterStoreContext } from "../types";
+
+/**
+ * @internal
+ */
+export type PromiseReturnType<T> = T extends Promise<infer U> ? U : any;
+
+/**
+ * @internal
+ */
+export type PromiseFnReturnType<T extends (...args: any) => any> = PromiseReturnType<ReturnType<T>>;
+
+/**
+ * @internal
+ */
+export function* getAttributeFilterContext(): SagaIterator<AttributeFilterStoreContext> {
+    return yield getContext("attributeFilterContext");
+}
+
+/**
+ * @internal
+ */
+export function* cancelableCall<T>({
```

Comment:
Typo: should be cancellable (two ls).

Reply:
I think that both cancelable / cancellable is correct. (US with one vs British with two Ls)

Reply:
Oh, ok, I see the existing hook is also one l :)